packages/mantine/src/menu/Menu.tsx

```diff
@@ -107,11 +109,22 @@ const SubMenu = forwardRef<
       }}>
       <Mantine.Menu.Item
         className="bn-menu-item bn-mt-sub-menu-item"
-        ref={ref}
+        ref={(node) => {
```

Comment:
you can use `const combinedRef = mergeRefs(ref, itemRef);` for this (from `@mantine/hooks`)